tools/settings.py

```diff
@@ -81,6 +81,9 @@
     'OPT_LEVEL',
     'DEBUG_LEVEL',
 
+    # Affects ports
+    'GL_ENABLE_GET_PROC_ADDRESS', # NOTE: if SDL2 is updated to not rely on eglGetProcAddress(), this can be removed
```

Comment:
I wonder if this should be called `ENABLE_GL_GET_PROC_ADDRESS`?    Not sure it that is better or worse.

Reply:
All GL settings have traditionally been prefixed in settings.py as `GL_xxx`. So I would not want to stray away from that convention. And `GL_ENABLE_GL_GET_PROC_ADDRESS` wouldn't look that good, hence condensing to just `GL_ENABLE_GET_PROC_ADDRESS`.